integration/tests/bft/protocol/disallowlisting/orchestrator.go

```diff
@@ -16,14 +16,15 @@ import (
 )
 
 const (
-	// numOfAuthorizedEvents number of events to send before blocking the sender node via the block list command.
-	numOfAuthorizedEvents = 10
+	// numOfAuthorizedEvents number of events to send before disallow-listing the sender node via the disallow-list command.
+	numOfAuthorizedEvents = 5
```

Comment:
What was the reason for reducing `numOfAuthorizedEvents`,  `numOfUnauthorizedEvents`? Was it to reduce test run times? Flakiness? It would be helpful if the reasoning for these values was more explicitly stated and to call out if there's a danger in making them too high.

Reply:
https://github.com/onflow/flow-go/pull/4652/commits/53a2434294c4240a01e813a8d23c0eaabdc17421